tests/integration/workflows/conftest.py

```diff
@@ -187,7 +187,7 @@ def mocked_external_services(workflow_app):
             requests_mock.ANY,
             re.compile(
                 '.*' +
-                workflow_app.config['BEARD_API_URL'] +
+                workflow_app.config['CLASSIFIER_API_URL'] +
```

Comment:
this is wrong, as the new classifier doesn't provide the `phonetic_blocks` service. On the other hand, I don't think it's used any longer, so you can just remove this mock.